src/to_makie.jl

```diff
@@ -32,6 +32,10 @@ function MakieCore.Theme(
                     titlefont = setting.font,
                     labelfont = setting.font,
                 ),
+                Colorbar = (
+                    labelfont = setting.font,
+                    ticklabelfont = setting.font,
+                )
```

Comment:
this might be the culprit for the failing formatting job
```suggestion
                ),
```